netbox/utilities/conversion.py

```diff
@@ -10,7 +10,7 @@
 )
 
 
-def to_grams(weight, unit):
+def to_grams(weight, unit) -> float:
```

Comment:
IMO it would make more sense for `to_grams()` to return an integer value. It's only used to populate integer `_abs_weight` fields and there doesn't seem to be any practical need to store fractional grams.

Reply:
Probably right; though cleaning up `to_grams` also suggests `to_meters` needs some attention to ensure it's always returning either a float or a Decimal consistently. Probably for a different (housekeeping) ticket though.